### PR TITLE
Pjacobsam/CHM306---migrate kmvalueterm to sbcd

### PR DESCRIPTION
**Pull request by pjacobsam**

### CODE REVIEW

File: src/components/cbd-nav/cbd.css
Comment (blaisefonseca) [inline src/components/cbd-nav/cbd.css:1]: why was this file deleted?

File: src/components/controls/view/km-value-term.vue [hunk 1]
```diff
@@ -0,0 +1,31 @@
+<template>
+  <div :id="$attrs.id" class="scbd-controls km-value-term">
+    <div :class="`input-group input-lang-${locale}`">
+      <div class="form-control km-value" :dir="direction(kmInitializeTerm && kmInitializeTerm.title, locale)">
```
Comment (blaisefonseca) [inline src/components/controls/view/km-value-term.vue]:
```suggestion
      <div class="form-control km-value" :dir="direction(kmInitializeTerm?.title, locale)">
```

File: src/components/controls/view/km-value-term.vue [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+<script setup>
+import { ref, defineProps } from 'vue';
+import { direction } from "../../../services/util/index"
+import KmTerm from './km-term.vue';
+
+const props = defineProps({
+  value: { type: Object, required: true },
+  locale: { type: String, required: true },
+});
+const kmInitializeTerm = ref(null);
```
Comment (blaisefonseca) [inline src/components/controls/view/km-value-term.vue:21]:
```suggestion
const kmInitializedTerm = ref(null);
```

Comment (blaisefonseca) [inline src/components/controls/view/km-value-term.vue]:
```suggestion
const termValue = ref(null);
```

File: src/components/cbd-nav/cbd.css
Comment (pjacobsam) [reply src/components/cbd-nav/cbd.css:1]: We have the same file in main under "src/assets/cbd.css"

**Review by blaisefonseca — APPROVED** [review]
